test/spec/provider/cloud-element-templates/fixtures/complex.json

```diff
@@ -683,5 +683,139 @@
         }
       }
     ]
+  },
+  {
```

Comment:
Let's not mangle it into `complex.json`, but instead keep it in a separate file, under a clear name, i.e. `update-properties-order.json`.

You can rely on all templates in `cloud-element-templates/fixtures` to be loaded by the test, cf. https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/test/spec/BpmnPropertiesPanelRenderer.spec.js#L245.